frontend/src/components/ProductCard.vue

```diff
@@ -37,31 +45,12 @@
         <v-icon>{{ show ? 'mdi-chevron-up' : 'mdi-chevron-down' }}</v-icon>
       </v-btn>
     </v-card-actions> -->
-
-    <v-expand-transition>
-      <div v-show="show">
-        <v-divider/>
-
-        <v-card-text>
```

Comment:
@PedroFW eu tive que remover essa parte aqui pq a frase ficava muito longa e o linter reclamava, como não estava sendo usado achei de boa remover, se for um problema só falar que eu restaura antes de fechar o PR.

Reply:
Tranquilo